src/app/cube/collections/collections.component.scss

```diff
@@ -70,26 +68,25 @@ $content-width: calc(#{$max-width} - #{$sidebar-width});
   }
 
   .header {
-    color: white;
+    color: $dark-grey;
     font-size: $largest;
     text-transform: uppercase;
     font-weight: bold;
-    text-shadow: 0 0 25px rgba(0, 0, 0, 0.5);
   }
 
   .description {
-    color: white;
+    color: $dark-grey;
     font-size: $large;
-    text-shadow: 0 0 25px rgba(0, 0, 0, 0.5);
+    text-shadow: 0 0 45px rgba(0, 0, 0, 0.5);
     font-weight: normal;
   }
 }
 
 .collections__grid {
-  max-width: $max-width;
+  max-width: 1600px;
```

Comment:
What was the max width before?

Reply:
max-width is 1400px

Reply:
Okay, lets just adjust the max-width to 1600px and keep using the variable

Reply:
Why should we change the width?  Sorry I'm a little lost.

Reply:
i changed the max -width so the cards can be stretch @EVogel1999

Reply:
@EVogel1999 I was just wondering why we didn't just change the variable I think it's defined at the top of the stylesheet.